src/applications/vaos/containers/ExpressCareFormPage.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { FETCH_STATUS } from '../utils/constants';
+import FormButtons from '../components/FormButtons';
+
+import * as actions from '../actions/expressCare';
+
+function ExpressCareFormPage({
+  submitStatus,
+  submitExpressCareRequest,
+  router,
+  routeToPreviousAppointmentPage,
+}) {
+  return (
+    <div>
+      <h1>Express Care form</h1>
+      <FormButtons
```

Comment:
This is a throw away page, the buttons and error alert should be carried over to whatever the real form page is.